server/unithand.cpp

```diff
@@ -3772,6 +3761,24 @@ static bool unit_bombard(struct unit *punit, struct tile *ptile,
   }
   unit_list_iterate_safe_end;
 
+  // Notify the players hit by the bombardment
+  for (auto player_to_notify : players_to_notify) {
+    notify_player(
+        player_to_notify, ptile, E_UNIT_BOMB_DEF, ftc_server,
+        /* TRANS: Your units in [tile] have been bombarded by the French
```

Comment:
'in' vs 'on'? Your units **on** [tile] have been bombarded by the [Nation] [Unit]. Unless the tile is a city, then "in" works, but [tile] would be a city name and not a map address.

Reply:
changed to "at"